minecraft_servers/mcshowcase/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "server_name": "mcshowcase",
+    "nice_name": "MCShowcase.net",
+    "direct_ip": "mcshowcase.net",
+    "social": {
+      "web": "https://mcshowcase.net",
```

Comment:
Maintenance; how long will it take? If it'll take some time, remove and add with another PR